ClassificationNeuralNetwork.java

```diff
@@ -1,8 +1,65 @@
-
+import java.io.*;
+import java.lang.Integer;
 
 public class ClassificationNeuralNetwork
 {
 	public static void main(String [] args)
 	{
 	}
+
+	public static void readDataFromFile(int [] classification,
+			double [] radii, double [] textures, double [] perimeters,
+			double [] area, double [] smoothness, double [] compactness,
+			double [] concavity, double [] concavePnts, double [] symmetry, double [] fractalDim)
+	{
+		String dataFileName = "data.txt";
+		String line = "";
+		int numLines = 569;
+
+		classification = new int[numLines];
+		radii = new double[numLines];
+		textures = new double[numLines];
+		perimeters = new double[numLines];
+		area = new double[numLines];
+		smoothness = new double[numLines];
+		compactness = new double[numLines];
+		concavity = new double[numLines];
+		concavePnts = new double[numLines];
+		symmetry = new double[numLines];
+		fractalDim = new double[numLines];
+		int currLineNum = 0;
+
+		try
+		{
+			FileReader fileReader = new FileReader(dataFileName);
+			BufferedReader bufferedReader 
```

Comment:
Just one suggestion: you could probably combine these two lines together as the following:

BufferedReader bufferedReader= new BufferedReader(new FileReader(dataFileName));

Reply:
Aside from that, it looks good.